scripts/install_backend_python_libs.py

```diff
@@ -28,12 +28,31 @@
 import pkg_resources
 
 
+def _normalize_string_that_contains_python_library_name(original_string):
```

Comment:
Don't call it original_string, that doesn't have conceptual meaning. The string is just the type. Same thing for normalized_string.

Try and figure out conceptually what is going on. original_library_name is probably the input, but I'm not clear on what exactly the output is. (For example, is it meant to represent a "directory name" or a "library name"? Etc. It's fine to add "normalized" before it, but it's important to be clear about what "it" is.)